packages/SketchMorph2-Core.package/M2Backend.class/instance/restoreBrushSize.color.isFilling.tool..st

```diff
@@ -0,0 +1,7 @@
+undo
+restoreBrushSize: aNumber color: aColor isFilling: aBoolean tool: aSymbol
+	self
+		brushSize: aNumber;
```

Comment:
What about the eraser size? Why didn't you user our method `setBrushAndEraserSize`?

Reply:
Good point.

Reply:
Done in the latest commits.